package.json

```diff
@@ -23,21 +23,21 @@
       "import": "./dist/js/uswds.min.js",
       "default": "./dist/js/uswds.min.js"
     },
-    "./src/js/components": "./src/components/index.js",
-    "./src/js/components/*": "./src/js/components/*.js",
-    "./src/fonts/*": "./src/fonts/*",
-    "./src/img/*": "./src/img/*",
-    "./js/*": "./dist/js/*",
+    "./src/js/components": "./packages/uswds-core/src/js/index.js",
+    "./src/js/components/*": "./packages/*.js",
+    "./src/fonts/*": "./packages/uswds-core/src/assets/fonts/*",
+    "./src/img/*": "./packages/usa-icon/src/img",
+    "./js/*": "./packages/*.js",
```

Comment:
Slightly better than the `src/js/components/*`, but still verbose

**Example**
```js
import accordion from "@uswds/uswds/js/usa-accordion/src/accordion";
```
---

💡

1. If we named `accordion.js → index.js`  
2. We could update package.json to export `    "./js/*": "./packages/*/src/index.js",`
3. Which would allow users to import `import accordion from "@uswds/uswds/js/usa-accordion";`

Reply:
I think that's a good idea!

Reply:
Done!